src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -316,7 +323,9 @@
 	public static final Item ICE_SHARD = getNull();
 	public static final Item HORN = getNull();
 	public static final Item CAKE_MIX = getNull();
-	
+	public static final Item QUEENS_RING_BLACK = getNull();
+	public static final Item QUEENS_RING_WHITE = getNull();
```

Comment:
I think white_queens_ring and black_queens_ring work better as names

Reply:
Same situation with the scepters, I believe they should stay like this

Reply:
Minecraft uses proper word order for their registry names (see any colored block), and I think we should do the same to have more predictable registry names